api/src/main/java/openconsensus/metrics/DerivedDoubleCumulative.java

```diff
@@ -30,12 +29,13 @@
  * <pre>{@code
  * class YourClass {
  *
- *   private static final MetricRegistry metricRegistry = Metrics.getMetricRegistry();
+ *   private static final MetricCollection metricCollection =
+ *       Metrics.getMeter().createMetricsCollection().build();
  *
  *   List<LabelKey> labelKeys = Arrays.asList(LabelKey.create("Name", "desc"));
  *   List<LabelValue> labelValues = Arrays.asList(LabelValue.create("Inbound"));
  *
- *   DerivedDoubleCumulative cumulative = metricRegistry.addDerivedDoubleCumulative(
```

Comment:
I like registry more. What's the rationale for renaming?

Reply:
I can keep the name registry. The only concern that I had was that Registry sounds a bit more global than collection. I would go with registry for the moment.